web/src/app/modules/shared/components/presentation/icon/icon.component.ts

```diff
@@ -51,6 +53,17 @@ export class IconComponent extends AbstractViewComponent<IconView> {
 
   protected update(): void {
     const view = this.v;
+    this.tooltip = view.config.tooltip;
```

Comment:
This function is already long it will be nice to have this functionality in another function